src/BlocksRuntime/BlocksRuntime.def

```diff
@@ -1,4 +1,4 @@
 LIBRARY BlocksRuntime
 EXPORTS
-  _NSConcreteGlobalBlock CONSTANT
-  _NSConcreteStackBlock CONSTANT
+  _NSConcreteGlobalBlock DATA
+  _NSConcreteStackBlock DATA
```

Comment:
Please don't change this.  It was using `CONSTANT` for a very specific reason - the generated import library is different.

Reply:
Apologies, that shouldn't have been in there. Was just experimenting with getting it to build.